lib/settingsGenerated.ts

```diff
@@ -24,7 +24,14 @@ export interface ISettings {
     'preferredLogicTypeSignal': 'unresolved'|'resolved'|'ignore';
     'unusedSignalRegex': string;
     'ieeeCasing': 'lowercase'|'UPPERCASE';
+    'signalPrefix': string;
+    'signalSuffix': string;
+    'genericPrefix': string;
+    'genericSuffix': string;
+    'instantiationLabelPrefix': string;
+    'instantiationLabelSuffix': string;
```

Comment:
Theses styles should also be enforced in a rule. Similarly, the in and out regex should be migrated to this style and enforced in the same way (with quick fixes)